util/jsonrpc-types/src/uints.rs

```diff
@@ -182,12 +182,6 @@ impl From<core::Capacity> for JsonUint<u64> {
     }
 }
 
-impl From<core::EpochNumberWithFraction> for JsonUint<u64> {
-    fn from(value: core::EpochNumberWithFraction) -> Self {
-        JsonUint(value.full_value())
```

Comment:
I don't understand why this is dangerous?

Reply:
```rust
// core_epoch: core::EpochNumberWithFraction

let epoch: jsonrpc_types::EpochNumber = core_epoch.into();
let epoch: jsonrpc_types::EpochNumberWithFraction = core_epoch.into();
```

I just worry that the user mix-uses transition from `core::EpochNumberWithFraction` to `EpochNumber` and `EpochNumberWithFraction`. Although lacking a strong reason, I shall close this PR. Thanks